lib/Target/CBackend/CBackend.cpp

```diff
@@ -1291,7 +1291,7 @@ void CWriter::printConstant(Constant *CPV, enum OperandContext Context) {
     if (Ty == Type::getInt1Ty(CPV->getContext())) {
       Out << (CI->getZExtValue() ? '1' : '0');
     } else if (Context != ContextNormal && ActiveBits < 64 &&
```

Comment:
```suggestion
    } else if (Context != ContextNormal &&
```

Reply:
I guess you are suggesting this because it's redundant given the other ActiveBits condition?

Reply:
Yeah, I assuming the conditions are actually related, so no sense in listing it twice?